CONTRIBUTING.md

```diff
@@ -1,63 +0,0 @@
-# Contributing to Parse Facebook Utils for Android
```

Comment:
Uhm? Shouldn’t we rewrite it?

Reply:
It links out to the main one now, since it is the one that is kept up to date:
https://github.com/parse-community/Parse-SDK-Android/blob/master/CONTRIBUTING.md

Reply:
Ok cool! Good idea to keep a single one. For iOS I merged all the repos a while ago to ease mainteanance. Is it something you’d like to do too?

Reply:
Ahh, like merged the Facebook and Twitter utils repos? And the UI repo?

Reply:
yep, I did that a while ago to consolidate CI configurations primarily.

Reply:
I think that is a good idea. I will go ahead and release this as 1.11.0 but I think that is a good next step for these repos 👍